eng/publish.proj

```diff
@@ -0,0 +1,89 @@
+<Project>
+  <Import Project="..\Directory.Build.props" />
+  <Import Project="..\Directory.Build.targets" />
+
+  <PropertyGroup>
+    <GitHubRepositoryName Condition="'$(GitHubRepositoryName)' == ''">corefx</GitHubRepositoryName>
+    <AssetManifestFilePath>$(ArtifactsDir)AssetManifests\$(GitHubRepositoryName).xml</AssetManifestFilePath>
+
+    <PackageOutputRoot Condition="'$(PackagesOutputRoot)' == ''">$(ArtifactsDir)/packages/</PackageOutputRoot>
+    <FinalPublishPattern>$(PackageOutputRoot)*.nupkg</FinalPublishPattern>
+    <FinalPublishPrivatePattern>$(PackageOutputRoot)*Private*.nupkg</FinalPublishPrivatePattern>
+    <FinalPublishExperimentalPattern>$(PackageOutputRoot)*Experimental*.nupkg</FinalPublishExperimentalPattern>
+  </PropertyGroup>
+
+  <ItemGroup Condition="'$(PackagesGlob)' != ''">
+    <PackagesToPublish Include="$(PackagesGlob)" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(PackagesGlob)' == ''">
+    <PackagesToPublish Include="$(FinalPublishPrivatePattern);$(FinalPublishExperimentalPattern)">
+      <ManifestArtifactData>NonShipping=true</ManifestArtifactData>
+    </PackagesToPublish>
+    <PackagesToPublish Include="$(FinalPublishPattern)" Exclude="@(PackagesToPublish)" />
+  </ItemGroup>
+
+  <Target Name="PublishToAzureBlobFeed">
```

Comment:
The one in arcade lacks of passing ManifestCommit and AssetsManifestPath parameters. Should we add that in arcade directly? Those differences are the reason why I didn't use that guy.

Reply:
Yes, that's busted.  I see ManifestComment (unless I'm looking in the wrong place) - https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Build.Tasks.Feed/build/Microsoft.DotNet.Build.Tasks.Feed.targets#L84

I don't see AssetsManifestPath, but it should be included.

I think @ChadNedzlek mentioned this a few days ago (or last week) but nobody has actually fixed it.  We should fix this in Arcade rather than requiring every team to re-implement these targets.

FYI @jcagme

Reply:
Also, it's possible that you're not seeing ManifestCommit in the Arcade restored version of the package feed because it's using an older version of the Feed package (defined in the Sdk's DefaultVersions.props file).  Just FYI

Reply:
I'll open an issue and not block this PR on that. Once that is fixed I'll send a follow up PR

Reply:
We restore the latest Feed package from Tools.props. So we're not using the one that comes built in from arcade.

Reply:
Thanks @safern .

@markwilkie @jcagme , it'd be great to get this fixed sooner than later.  I have a feeling that the damage is (mostly) already done though as I'm pretty sure that a lot of folks are just writing their own targets (making it hard to update anything without breaking people).  I've heard @ChadNedzlek @natemcmaster and now @safern mention this though I hadn't connected Nate's comments until just now.

Reply:
The issue already exists: https://github.com/dotnet/arcade/issues/1555